Documentation/development-environment.md

```diff
@@ -158,16 +158,14 @@ subjects:
 
 3) Find the admin-user token in the kube-system namespace:
 
-```bash
+```console
```

Comment:
what's the difference between bash and console in the rendering?

Reply:
At least on GitHub, possibly also for our Jekyll markdown rendering, it makes commands look like https://stackoverflow.com/a/49004070